service_mesh/v2x/ossm-gateway-migration.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-gateway-migration"]
+= Gateway Migration
+include::_attributes/common-attributes.adoc[]
+:context: gateway-migration
+
+toc::[]
+
+As a network administrator, the preferred method for deploying ingress and egress gateways is to do so with a `Deployment` resource using gateway injection.
+
+include::modules/ossm-about-gateway-migration.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-from-smcp-defined-gateways-to-gateway-injection.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
So is this additional resources section meant to apply to the entire assembly, or just the module that this is directly underneath? If it's for the assembly, this is how it should be formatted:
```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```

If it's just meant for the module above it, then all it needs is a role attribute:
```suggestion
[role="_additional-resources"]
.Additional resources
```

Source: [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)

Reply:
updated for the assembly. done.